pypi_formula_mappings.json

```diff
@@ -31,7 +28,7 @@
       "zabbix-api", "junos-eznc", "jxmlease", "dnspython", "pysphere3", "python-consul", "requests-credssp",
       "openshift", "pexpect", "ntc-templates", "proxmoxer"
     ],
-    "exclude_packages": ["certifi", "cffi", "pycparser", "PyYAML", "six"]
+    "exclude_packages": ["certifi"]
```

Comment:
This is for `ansible@7`, which we disabled in https://github.com/Homebrew/homebrew-core/pull/159565. Everything else looks good to me.

Reply:
Womp, thanks. Must have been left over from me starting with `@7` and then stopping when I realized it was disabled.